src/lib/env/migrationsDb.ts

```diff
@@ -149,3 +137,29 @@ export async function getAllMigrations() {
     await recursiveProcess();
     return migrations;
 }
+
+function loadConfig(inputProfile: string) {
+    const configFromFile = config.readConfig();
+
+    // Check for data for input profile
+    const profileConfig = configFromFile.find((obj: any) => {
+        return obj.profile === inputProfile || (!obj.profile && inputProfile === 'default');
+    });
+
+    // Populate  region
+    if (profileConfig && profileConfig.region) {
+        AWS.config.region = profileConfig.region;
+    } else {
+        throw new Error(`Please provide region for profile:${inputProfile}`);
+    }
+
+    if (profileConfig && profileConfig.accessKeyId && profileConfig.secretAccessKey) {
+        AWS.config.update({
+            accessKeyId: profileConfig.accessKeyId,
```

Comment:
If a profile is provided, there is no need to specify accessKeyId and secretAccessKey. It should get the profile from `~/.aws/credentials`.

Reply:
Yes, if not provided by the user in config, it gets the details from ~/.aws/credentials. However if the users provide the accessKey and secretAccessKey in config then the same takes precedence over ~/.aws/credentials.